### PR TITLE
Dubba175

### PR DESCRIPTION
**Pull request by drhigsby**
Dubba175 is a keyboard that I wish to merge with QMK primary

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x ] Keyboard (addition or update)
- [x ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/dubba175/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+#include "quantum.h"
```
Comment (drashna) [inline keyboards/dubba175/keymaps/default/keymap.c:17]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/dubba175/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+enum layers{
+  _BASE,
+  _NUM,
+  _NAV,
+  _FN
+
```
Comment (ridingqwerty) [inline keyboards/dubba175/keymaps/default/keymap.c:24]:
```suggestion
```

File: keyboards/dubba175/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+combo_t key_combos[COMBO_COUNT] = {
+  [combo_ESC] = COMBO(esc_combo, KC_ESC),
+   [combo_BACK] = COMBO(bspc_combo, KC_BSPC),
+    [combo_TAB] = COMBO(tab_combo, KC_TAB),
+     [combo_DELETE] = COMBO(del_combo, KC_DEL),
+      [combo_ENTER] = COMBO(enter_combo, KC_ENT),
+       [combo_QUOT] = COMBO(quot_combo, KC_QUOT),
+        [combo_LPRN] = COMBO(lprn_combo, KC_LPRN),
+         [combo_RPRN] = COMBO(rprn_combo, KC_RPRN),
```
Comment (ridingqwerty) [inline keyboards/dubba175/keymaps/default/keymap.c:56]:
```suggestion
  [combo_ESC] = COMBO(esc_combo, KC_ESC),
  [combo_BACK] = COMBO(bspc_combo, KC_BSPC),
  [combo_TAB] = COMBO(tab_combo, KC_TAB),
  [combo_DELETE] = COMBO(del_combo, KC_DEL),
  [combo_ENTER] = COMBO(enter_combo, KC_ENT),
  [combo_QUOT] = COMBO(quot_combo, KC_QUOT),
  [combo_LPRN] = COMBO(lprn_combo, KC_LPRN),
  [combo_RPRN] = COMBO(rprn_combo, KC_RPRN),
```

File: keyboards/dubba175/rules.mk [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = caterina
+
+BOOTMAGIC_ENABLE = no      # Virtual DIP switch configuration
```
Comment (ridingqwerty) [inline keyboards/dubba175/rules.mk:14]:
```suggestion
BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
```

File: keyboards/dubba175/keymaps/default/config.h [hunk 5]
```diff
@@ -0,0 +1,19 @@
+/* Copyright 2020 tominabox1
```
Comment (ridingqwerty) [inline keyboards/dubba175/keymaps/default/config.h:1]: "tominabox1"?

**Review by ridingqwerty — CHANGES_REQUESTED** [review]

File: keyboards/dubba175/keymaps/default/config.h (hunk 5 above)
Comment (drhigsby) [reply keyboards/dubba175/keymaps/default/config.h:1]: Apologies. I was looking at tominabox1's le_chiffre QMK branch for help understanding what changes I needed to make to dubba175 before submission.

Comment (drhigsby) [reply keyboards/dubba175/keymaps/default/config.h]: I have committed a change for it to read "drhigsby"

**Review by drashna — APPROVED** [review]

File: keyboards/dubba175/readme.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
+# Dubba175
+
+![Dubba175](https://imgur.com/gallery/yryBnVg)
```
Comment (fauxpark) [inline keyboards/dubba175/readme.md:3]:
```suggestion
![Dubba175](https://i.imgur.com/nzDZuS3h.jpg)
```
Needs to be a direct URL.

File: keyboards/dubba175/rules.mk [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
```
Comment (fauxpark) [inline keyboards/dubba175/rules.mk:11]:
```suggestion
```

File: keyboards/dubba175/rules.mk [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+BOOTLOADER = caterina
+
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+NKRO_ENABLE = yes           # USB Nkey Rollover
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+LTO_ENABLE = no             # Use link time optimization
+TAP_DANCE_ENABLE = no
```
Comment (fauxpark) [inline keyboards/dubba175/rules.mk:21]:
```suggestion
# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
LTO_ENABLE = no             # Use link time optimization
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drhigsby** [comment]
Is there anything further I need to do to help out with the PR?

**Review by ridingqwerty — APPROVED** [review]
Looks good to me.

**Comment by ridingqwerty** [comment]
Thanks!

**Pull request merged**